src/gleam/string.gleam

```diff
@@ -176,26 +176,34 @@ pub fn contains(does haystack: String, contain needle: String) -> Bool {
   erl_contains(haystack, needle)
 }
 
-// TODO
-// TODO: Not sure about the name and labels here
-// See if the second string starts with the first one.
-//
-// ## Examples
-//    > starts_with(does: "theory", start_with: "ory")
-//    False
-//
-//
-// pub fn starts_with(does string: String, start_with prefix: String) -> String {}
-// TODO
-// TODO: Not sure about the name and labels here
-// See if the second string ends with the first one.
-//
-// ## Examples
-//    > ends_with(does: "theory", end_with: "ory")
-//    True
-//
-//
-// pub fn ends_with(does string: String, end_with suffix: String) -> String {}
+external fn erl_prefix(String, String) -> Bool =
+  "gleam_stdlib" "string_starts_with"
+
+/// See if the first string starts with the second one.
+///
+/// ## Examples
+///    > starts_with("theory", "ory")
+///    False
+///
+///
+pub fn starts_with(string: String, prefix: String) -> Bool {
+  erl_prefix(string, prefix)
```

Comment:
I think we can export the external functions directly, they accept the arguments in the order we want